packages/jaeger-ui/src/components/TracePage/CriticalPath/index.test.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TraceCriticalPath, {
+  findRootSpanId,
+  findChildSpanIds,
+  findLastFinishingChildSpanId,
+  computeCriticalPath,
+} from './index';
+import transformTraceData from '../../../model/transform-trace-data';
+import criticalPathSections from './testResults/test1';
+
+const testTrace = require('../TraceStatistics/tableValuesTestTrace/testTraceNormal.json');
+
+const transformedTrace = transformTraceData(testTrace);
+
+const defaultProps = {
+  trace: transformedTrace,
+};
+
+describe('Happy Path', () => {
```

Comment:
Let's keep the unit-tests for util methods(e.g. `findRootSpanId`,`findChildSpanIds`) etc. under a separate `describe` group. If you checkout the docs for jest, you would see that we generally use `describe` for grouping tests which are logically related.

I would logically break down the tests for - 
* Helper Methods
* Specific Unit Tests

Reply:
Yepp ,I separated each helper functions and unit tests in describe blocks. let me know if I should do any changes